src/stratis_cli/_error_reporting.py

```diff
@@ -151,6 +151,8 @@ def _interpret_errors(errors):
             )
             return fmt_str % error
 
+        if isinstance(error, BrokenPipeError):
```

Comment:
You didn't add a test for this branch and when the "no cover" pragma is removed when work for https://github.com/stratis-storage/stratis-cli/issues/371 is completed, this code will be exposed as uncovered. I don't think we will ever end up putting a test in the CI for this, so the thing to do is to explain why it is not tested (and add the "no cover" pragma after the if-expression).